fxgl-ai/src/test/java/com/almasb/fxgl/pathfinding/GridTest.java

```diff
@@ -73,7 +91,7 @@ public void testGetRandomCell() {
     @Test
     public void testRandomCell() {
         for (int i = 0; i < 50; i++) {
-            Optional<MockCell> c = grid.getRandomCell(cell -> cell.getX() < 3 && cell.getY() > 15);
+            Optional<MockCell> c = grid.getRandomCell(cell -> cell.getX() < 3 && cell.getY() > CELL_HEIGHT);
```

Comment:
Please revert the change to this line. `15` here is not `CELL_HEIGHT`, even though the value is the same. Otherwise, a change of `CELL_HEIGHT` may unintentionally break this test.

Reply:
Oops, sorry, thats was unintended, fixed